### PR TITLE
Add Wikis: Linkin Park, Russian Lethal Company, & Combine OverWiki Redirects (Russian, Spanish, Chinese, Polish)

### PR DESCRIPTION
**Pull request by DrWhoFan13**
Miraheeze has a complete Russian translation of the Lethal Company Wiki, which also has a Fandom wiki of 72 pages. This Pull Request has a redirect for that wiki.

Update: I've since added Redirects for all compatible Half-Life Fandoms and fixed an error in the LC Russian Redirect.

### CODE REVIEW

File: data/sitesES.json [hunk 1]
```diff
@@ -35,6 +35,9 @@
     "destination_main_page": "Portada",
     "destination_search_path": "/index.php"
   },
+  {
+    "id": "
```
Comment (KevinPayravi) [inline data/sitesES.json:39]: Stray object here, please remove

**Review by KevinPayravi — CHANGES_REQUESTED** [review]

**Comment by DrWhoFan13** [comment]
Done!

**Comment by DrWhoFan13** [comment]
@KevinPayravi

File: data/sitesPL.json [hunk 2]
```diff
@@ -21,6 +21,25 @@
...
       "official"
     ]
+  },
+    {
+    "id": "pl-halflife",
+    "origins-label": "Half-Life Fandom Wiki",
+      "origins": [
+        {
+        "origin": "Half-Life Fandom Wiki",
+        "origin_base_url": "halflife.fandom.com/pl",
```
Comment (KevinPayravi) [inline data/sitesPL.json:31]: Looks like this URL should be `half-life`, not `halflife`

**Comment by KevinPayravi** [comment]
@DrWhoFan13 Apologies for the wait. With these redirects between non-English wikis that utilize the "destination_content_suffix", I've realized there are some issues, depending on the wiki. The redirects only really work when the non-English title is the same as the English one, since on the destination wiki, the language code (e.g. `/es`) is appended to the English title of the article. I've been trying to figure out a good way to gracefully handle when the titles do not match.

**Comment by DrWhoFan13** [comment]
So, this isn't gonna be merged unless I remove the code for the Lethal Company Redirect due to the aforementioned issue?

**Comment by DrWhoFan13** [comment]
Oh, so it is. I fixed the error.

**Comment by KevinPayravi** [comment]
> So, this isn't gonna be merged unless I remove the code for the Lethal Company Redirect due to the aforementioned issue?

I'll likely still merge it before the next update, just investigating if there's a better approach to handle these wikis

**Comment by DrWhoFan13** [comment]
> I'll likely still merge it before the next update, just investigating if there's a better approach to handle these wikis
Glad to hear it!

**Review by KevinPayravi — APPROVED** [review]

**Pull request merged**